nautobot/ipam/tests/test_views.py

```diff
@@ -40,6 +40,32 @@
 from nautobot.virtualization.models import Cluster, ClusterType, VirtualMachine
 
 
+class NamespaceTestCase(
+    ViewTestCases.GetObjectViewTestCase,
+    ViewTestCases.GetObjectChangelogViewTestCase,
+    ViewTestCases.GetObjectNotesViewTestCase,
+    ViewTestCases.CreateObjectViewTestCase,
+    ViewTestCases.EditObjectViewTestCase,
+    ViewTestCases.DeleteObjectViewTestCase,
+    ViewTestCases.ListObjectsViewTestCase,
+    ViewTestCases.BulkImportObjectsViewTestCase,
```

Comment:
Can we use `PrimaryObjectViewTestCase` instead of these mixins? Or do `BulkEdit` and `BulkDelete` tests not apply to Namespaces for some reason? (And if that's the case, should we change that?)

Reply:
Namespace do not currently support BulkEdit/BulkDelete, and fixing that would be out of scope for this issue

Reply:
Ok, let's get a follow-up issue opened then.

Reply:
https://github.com/nautobot/nautobot/issues/4272